pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -667,18 +667,20 @@ func (dm *DockerManager) runContainer(
 	}
 
 	// Set sysctls if requested
-	sysctls, unsafeSysctls, err := api.SysctlsFromPodAnnotations(pod.Annotations)
-	if err != nil {
-		dm.recorder.Eventf(ref, api.EventTypeWarning, events.FailedToCreateContainer, "Failed to create docker container %q of pod %q with error: %v", container.Name, format.Pod(pod), err)
-		return kubecontainer.ContainerID{}, err
-	}
-	if len(sysctls)+len(unsafeSysctls) > 0 {
-		hc.Sysctls = make(map[string]string, len(sysctls)+len(unsafeSysctls))
-		for _, c := range sysctls {
-			hc.Sysctls[c.Name] = c.Value
+	if container.Name == PodInfraContainerName {
```

Comment:
I have validated the expected behaviour now by replacing the kubelet manually on a 1.3 node with the binary of 1.4 (i.e. without node reboot/restart). Containers go through admission and are killed with the `SysctlUnsupported` reason.

Hence, while I still think we can merge this PR because the infra ctr is enough, it is not a security issue anymore, and not critical for 1.4.